BL_table/BL_table.Rmd

```diff
@@ -0,0 +1,259 @@
+---
+title: "Make a baseline table"
+author: "Author"
+date: "13.5.2022"
+output:
+  github_document: default
+bibliography: references_BL.bib
+editor_options: 
+  markdown: 
+    wrap: sentence
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, 
+                      warning = FALSE,
+                      message= FALSE)
+options(huxtable.knitr_output_format="md")
+```
+
+# Background
+
+Epidemiologic and clinical research papers often describe the study sample in a first, baseline table.
+Providing a baseline table is one of the recommendations of the CONSORT (@MOHER201228) and STROBE (@VONELM20071453) statements. If well-executed, it provides a rapid, objective, and coherent grasp of the data and can illuminate potential threats to internal and external validity (@HAYESLARSON2019125).
+
+
+
+# Main important considerations
+
+Baseline tables can be presented in different ways, but certain features are common and some recommendations need be followed. In the next paragraph and figure we provide an overview of the main important considerations, for more detailed review and guidelines see (e.g. @MOHER201228, @HAYESLARSON2019125).
+
+Characteristics are generally presented on the left as rows and groupings at the top as columns.
+Groups are defined by treatment allocation for RCTs, disease status for case--control studies or cohorts.If appropriated, the overall population' characteristics may be presented in a separate column.
+Characteristics should include of study participants (e.g., demographic, clinical, social) and information on exposures and potential data confounders.
+Inside each cell, descriptive statistics are typically given as n (%) for categorical variables and mean (standard deviation) or median (25th-75th percentile or minimum-maximum) for continuous variables.The type of statistical measurement used should be indicated (e.g., together with the characteristic, in a footnote).The number of participants with missing data should be reported for each variable of interest.
```

Comment:
n (%) may be called "frequency and percentage".
25th-75th percentile is usually called inter-quartile range (although you can give the range as such, or the two quartiles...)

Reply:
I'm maybe wrong but for me the interquartile range is the range (75th-25th percentile)